Add dc_burns api docs (#443)
helium/docs: docs/api/blockchain/dc_burns.mdx

---
id: dc_burns
sidebar_label: DC Burns
slug: /api/blockchain/dc_burns
---

import Tabs from "@theme/Tabs";
import TabItem from "@theme/TabItem";

# DC Burns

---

## DC Burn Totals

```
GET https://api.helium.io/v1/dc_burns/sum
```

> Returns the total Data Credits (DC) burntfor the whole network in a given

Reviewer: ```suggestion
Returns the total Data Credits (DC) burned for the whole network in a given
```

timeframe. Timestamps can be given in ISO 8601 format (e.g.

> `2020-08-27T00:00:00Z`) or in relative time (e.g. `-1 week`, which when url

Reviewer: ```suggestion
`2020-08-27T00:00:00Z`) or in relative time (e.g. `-1 week`, which, when url-
```

> esacped becomes `-1%20week`). When ommitted the current time (UTC) is assumed.

Reviewer: ```suggestion
escaped becomes `-1%20week`). When omitted the current time (UTC) is assumed.
```

The results can also be bucketed in time by specifying a `bucket` query
parameter which buckets information per bucket in the given timeframe. Data is
bucketed by time. Valid bucket values include `hour`, `day` and `week`).

For example to get the last 7 days of DC burns bucketed by day use the following

> path/parameters: `?min_time=-7%20day&bucket=day`

Reviewer: ```suggestion
path/parameters: 

`?min_time=-7%20day&bucket=day`
```

  }
}
```

</TabItem>
</Tabs>

---

## DC Burn Stats

```
GET https://api.helium.io/v1/dc_burns/stats
```

Returns current statics for for Data Credits (DC) burntfor the whole network.

<Tabs
  block={true}
  defaultValue="request"
  values={[{"label":"Request","value":"request"},{"label":"Response","value":"response"}]}>
<TabItem value="request">

_No Parameters_

</TabItem>
<TabItem value="response">

200: OK

Return the current DC burn stats

```json
{
  "data": {
    "last_day": {
      "add_gateway": 4660000000,
      "assert_location": 1486000000,
      "fee": 245860000,
      "state_channel": 1150548,
      "total": 6393010548
    },
    "last_month": {
      "add_gateway": 100348000000,
      "assert_location": 35019000000,
      "fee": 5784740000,
      "oui": 90000000,
      "state_channel": 31932182,
      "total": 141273672182
    },
    "last_week": {
      "add_gateway": 25060000000,
      "assert_location": 8483000000,
      "fee": 1425110000,
      "state_channel": 9392114,
      "total": 34977502114
    }
  },
  "meta": {
    "timestamp": "2021-06-25T21:10:14Z"
  }
}
```

</TabItem>
</Tabs>

---

## DC Burn Events

```
GET https://api.helium.io/v1/dc_burns
```

> List DC burn events. Events include a block, amount, type of burn and the

Reviewer: ```suggestion
Use this API to list DC burn events. Events include a block, amount, type of burn, and the
```

address of the account the DC were burned out of. The oracle price that was
active at the point the DC burn happened is also listed to enable calculating

> how much HNT the burnt DC represent.

Reviewer: ```suggestion
how much HNT the burned DC represents.
```
